packages/components/tree/src/tree.type.ts

```diff
@@ -63,7 +63,7 @@ export declare interface TreeOptionProps {
   children?: string
   label?: string | ((data: TreeNodeData, node: Node) => string)
   disabled?: string | ((data: TreeNodeData, node: Node) => string)
-  isLeaf?: boolean | ((data: TreeNodeData, node: Node) => string)
+  isLeaf?: string | ((data: TreeNodeData, node: Node) => string)
```

Comment:
`((data: TreeNodeData, node: Node) => string)` should return boolean?

Reply:
I think this should be `boolean | ((data: TreeNodeData, node: Node) => boolean)` because `isLeaf` sounds boolean to me.